qcodes/instrument_drivers/QuantumDesign/DynaCoolPPMS/DynaCool.py

```diff
@@ -0,0 +1,281 @@
+from qcodes.instrument.visa import VisaInstrument
+from functools import partial
+import logging
+from typing import Dict, Optional, Union, cast, Any, List
+import warnings
+
+from visa import VisaIOError
+
+import qcodes.utils.validators as vals
+
+log = logging.getLogger(__name__)
```

Comment:
This should probably be self.log after @Dominik-Vogel added the new logging api

Reply:
yes, we should remember to use the instrument specific logger in new drivers. Once I find time I'll go and convert old drivers.

Reply:
Changed that now.